addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -538,6 +554,26 @@ $o-bg-shapes: ('web_editor': (
     'Bold/11_001': ('position': bottom, 'size': 100% auto, 'colors': (1, 2)),
     'Bold/12': ('position': center, 'size': 100% auto, 'colors': (1, 2, 5)),
     'Bold/12_001': ('position': center, 'size': 100% auto, 'colors': (1, 2, 5)),
+    'Connections/01': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/02': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/03': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/04': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/05': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/06': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/07': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/08': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/09': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/10': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/11': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/12': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/13': ('position': bottom, 'size': 100%, 'colors': (5)),
+    'Connections/14': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/15': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/16': ('position': bottom, 'size': 100% var(--ConnectionShape--height-regular), 'colors': (5)),
+    'Connections/17': ('position': bottom, 'size': var(--ConnectionShape--height-short) 100%, 'colors': (5), 'repeat-x': true),
```

Comment:
Wait... but here the variable doesn't act on background "height", it acts on its "width".
I've  [suggested "height" here](https://github.com/odoo/odoo/pull/185342#discussion_r1846179388),  but because I've miss this specific scenario... have you as well? 

Ok, we should take back "size" wording I guess... but adapting the last part in order to avoid matching breakpoint names 👍 
```scss
--ShapeConnection--size-tiny
--ShapeConnection--size-regular
--ShapeConnection--size-big
```

Reply:
Indeed, that's probably why I used "size" first, sorry I should have seen it before 🤦 .